benchmark/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+PROJECT=acmeair-monolithic-java
+ACMEAIR_REPO=https://github.com/vpa1977/${PROJECT}
+
+git clone $ACMEAIR_REPO || \
```

Comment:
could you please create a `tag` in the upstream repo (or at least pin a commit) so that your upstream HEAD can move on without affecting this benchmark